src/bika/cement/browser/controlpanel/mixmaterialfolder.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2023 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+import collections
+
+from bika.cement.config import _
+from bika.lims import api
+from bika.lims.utils import get_link, get_link_for
+from senaite.app.listing import ListingView
+from senaite.core.catalog import SETUP_CATALOG
+
+
+class MixMaterialFolderView(ListingView):
+    """Displays all available sample containers in a table
+    """
+
+    def __init__(self, context, request):
+        super(MixMaterialFolderView, self).__init__(context, request)
+
+        self.catalog = SETUP_CATALOG
+
+        self.contentFilter = {
+            "portal_type": "MixMaterial",
+            "sort_on": "sortable_title",
+        }
+
+        self.context_actions = {
+            _("Add"): {
+                "url": "++add++MixMaterial",
+                "icon": "++resource++bika.lims.images/add.png",
+            }}
+
+        t = self.context.translate
+        self.title = t(_("Mix Materials"))
+        self.description = t(_(""))
+
+        self.show_select_column = True
+        self.pagesize = 25
+
+        self.columns = collections.OrderedDict((
+            ("title", {
+                "title": _("Title"),
+                "index": "sortable_title"}),
+            ("materialType", {
```

Comment:
Please also use lower casing for all the dict keys i.e instead of specificGravity go with specific_gravity.